cli/enry/main.go

```diff
@@ -32,24 +32,30 @@ func main() {
 		log.Fatal(err)
 	}
 
-	errors := false
+	fileInfo, err := os.Stat(root)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if fileInfo.Mode().IsRegular() {
+		printFileAnalysis(root)
+		os.Exit(0)
```

Comment:
The only point where the main function is exiting with a return is here, the other `return nil` are exiting the iteration in the Walk function https://golang.org/pkg/path/filepath/#Walk

Anyway, you can exit with `return nil` from a function which doesn't have return values declared in the signature.